lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -29,11 +29,11 @@
     </header>
 
     <ul class="wrapper-tabs">
-        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><%- gettext("Manage Students") %></a></li>
+        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><span class="sr"><%- gettext('Selected tab') %> </span><%- gettext("Manage Students") %></a></li>
```

Comment:
Isn't there a potential problem here for right-to-left languages? I know that when @talbs did this on another PR he made the entire section into a translated string so that the extra message for the screen reader didn't have to come first.

e.g. This one: https://github.com/edx/edx-platform/blob/cohorted-courseware/lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore#L77

@sarina Is this a concern?